src/lib/util/filterByCategory.ts

```diff
@@ -0,0 +1,26 @@
+import { Items } from 'oldschooljs';
+
+import { ItemTuple } from '../types';
+import { stringMatches } from '../util';
+import filterables from '../filterables';
+
+export default function filterByCategory(filterQuery: string, items: ItemTuple[]) {
+	const filteredBank: ItemTuple[] = [];
+	const filtered = filterables.find(
+		filtered =>
+			stringMatches(filtered.name, filterQuery) ||
+			stringMatches(filtered.name.split(' ')[0], filterQuery)
```

Comment:
the `|| ...` second check isnt really needed, none of the filterables are more than 1 word